<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>418</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>TODO List</string>
  </property>
  <property name="windowIcon">
   <iconset resource="To-Do-List-Manager.qrc">
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>381</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="addTask_button">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>320</y>
      <width>187</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Pridaj úlohu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editTask_button">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>320</y>
      <width>187</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Uprav úlohu</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="removeTask_button">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>320</y>
      <width>187</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Vymaž úlohu</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>191</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addCategory_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Pridaj kategóriu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeCategory_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Odstráň kategóriu</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>50</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Úloha:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>90</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Čas:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>130</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dokončená:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>170</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Poznámky:</string>
    </property>
   </widget>
   <widget class="QLabel" name="task">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>50</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="time">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>90</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="done">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>130</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="notes">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>200</y>
      <width>301</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>524</x>
      <y>-1</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri Light</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dokončená</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri Light</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Úloha</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri Light</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Kategória</string>
    </property>
   </widget>
   <widget class="QPushButton" name="login_button">
    <property name="geometry">
     <rect>
      <x>872</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Prihlásiť sa</string>
    </property>
   </widget>
   <zorder>addTask_button</zorder>
   <zorder>editTask_button</zorder>
   <zorder>removeTask_button</zorder>
   <zorder>addCategory_button</zorder>
   <zorder>removeCategory_button</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>task</zorder>
   <zorder>time</zorder>
   <zorder>done</zorder>
   <zorder>notes</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>tableView</zorder>
   <zorder>tableView_2</zorder>
   <zorder>login_button</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1015</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Súbor</string>
    </property>
    <addaction name="actionHide"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Nápoveda</string>
    </property>
    <addaction name="actionReport_A_Bug"/>
    <addaction name="actionDonate"/>
    <addaction name="separator"/>
    <addaction name="actionLicense"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Nástroje</string>
    </property>
    <addaction name="actionCheck_For_Update"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Ukončiť</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionReport_A_Bug">
   <property name="text">
    <string>Nahlásiť chybu</string>
   </property>
  </action>
  <action name="actionCheck_For_Update">
   <property name="text">
    <string>Aktualizuj program</string>
   </property>
  </action>
  <action name="actionDonate">
   <property name="text">
    <string>Podporte nás</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="text">
    <string>Licencia</string>
   </property>
  </action>
  <action name="actionHide">
   <property name="text">
    <string>Skryť do lištičky</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="To-Do-List-Manager.qrc"/>
 </resources>
 <connections/>
</ui>
